cmd/snap/cmd_create_user.go

```diff
@@ -36,6 +37,8 @@ An account can be setup at https://login.ubuntu.com.
 `)
 
 type cmdCreateUser struct {
+	Yaml       bool `long:"yaml" description:"output results in YAML format"`
```

Comment:
Slightly worried about the precedent here. The point of this is to offer a command line interface to the API. We commonly use progress bars and other textual output as we assume we're talking to a human being.

create-user is likely an exception as it's so clearly motivated by the needs of another tool, so let's go ahead, but there's a chance we may need to review this and move the client to use the actual API instead.

Let's please use JSON here, though. The rationale is this: if the content is intended to be consumed or produced by a human, we use yaml. If it's for cross-application communication, we use JSON as it's more widely available and a much simpler spec.